docs/libraries/workflow/GLOSSARY.md

```diff
@@ -0,0 +1,56 @@
+TODO
+
+# Glossary
+
+## Worker
+
+A process that's running workflows.
+
+There are usually multiple workers running at the same time.
+
+## Workflow
```

Comment:
I think putting Workflow before Worker would make more sense for the order of understanding ideas